code/obj/item/paper.dm

```diff
@@ -239,11 +239,10 @@
 				// the javascript was modified, somehow, outside of
 				// byond.  but right now we are logging it as
 				// the generated html might get beyond this limit
-				logTheThing("PAPER: [key_name(ui.user)] writing to paper [name], and overwrote it by [paper_len-PAPER_MAX_LENGTH]")
+				logTheThing("debug", src, null, "PAPER: [key_name(ui.user)] writing to paper [name], and overwrote it by [paper_len-PAPER_MAX_LENGTH]")
 			if(paper_len == 0)
 				boutput(ui.user, pick("Writing block strikes again!", "You forgot to write anthing!"))
 			else
-				logTheThing("PAPER: [key_name(ui.user)] writing to paper [name]")
```

Comment:
why remove this

Reply:
All the relevant information is included in the TGUI logs. I don't mind keeping it, but it feels redundant.